scripts/start.sh

```diff
@@ -103,14 +107,15 @@ echo Starting GAE development server
 
 if ! [[ "$FORCE_PROD_MODE" == "True" ]]; then
   ($NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js watch)&
+  ($NODE_MODULE_DIR/webpack/bin/webpack.js --config webpack.dev.config.js --watch)&
```

Comment:
Is this change intentional?

Reply:
No, removed.

Reply:
@vojtechjelinek, has this been reverted?